ts/util/legend.ts

```diff
@@ -1,14 +1,15 @@
 import { qcStatuses } from "../data/qc-status";
 import { makeIcon } from "./html-utils";
 
+export const legendID = "legend-container";
```

Comment:
* Should be `legendId` for consistency with other ID variable names throughout Dimsum
* No need to export